pontoon/base/models/user.py

```diff
@@ -220,6 +224,49 @@ def has_approved_translations(self):
     return self.translation_set.filter(approved=True).exists()
 
 
+@property
+def badges_translations_count(self):
+    """Contributions provided by user that count towards their badges."""
+    from pontoon.actionlog.models import ActionLog
+
+    return ActionLog.objects.filter(
+        performed_by=self,
+        action_type="translation:created",
+        created_at__gte=badges_inception,
+    ).count()
```

Comment:
I think this is clearer, since it's called on the `user` instance:

```suggestion
    return self.actions.filter(
        action_type="translation:created",
        created_at__gte=badges_inception,
    ).count()
```